CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Transport: Remove duplicate call to method (PR #931).
 * RTCP: Adjust maximum compound packet size (PR #934).
 * Fix `bufferedAmount` type to be a number again (PR #936).
+* SctpAssociation: fix memory violation (PR #943).
```

Comment:
OMG this is being so hard. Can we normalize those changelog entries to have capital letter after the colon and use code text in class names?

```
`SctpAssociation`: Fix memory violation (PR #943).
```